storage/sectorblocks/blocks.go

```diff
@@ -33,16 +29,28 @@ const (
 )
 
 var dsPrefix = datastore.NewKey("/sealedblocks")
-var imBlocksPrefix = datastore.NewKey("/intermediate")
 
 var ErrNotFound = errors.New("not found")
 
+func DealIDToDsKey(dealID uint64) datastore.Key {
+	buf := make([]byte, binary.MaxVarintLen64)
+	binary.PutUvarint(buf, dealID)
```

Comment:
probably want to catch the return value of PutUVarint and use it to slice the buf before making it a key